GooglePlayInstant/Editor/PlayInstantQuickDeployWindow.cs

```diff
@@ -0,0 +1,123 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using UnityEditor;
+using UnityEngine;
+
+namespace GooglePlayInstant.Editor
+{
+    public class PlayInstantQuickDeployWindow : EditorWindow
+    {
+        private static int toolbarInt = 0;
+        private string[] toolbarStrings = new string[] {"Bundle Create", "Bundle Deploy", "Verify Bundle", 
+            "Loading Screen", "Build"};
+
+        public const int BundleCreateSelect = 0;
+        public const int BundleDeploySelect = 1;
+        public const int VerifyBundleSelect = 2;
+        public const int LoadingScreenSelect = 3;
+        public const int BuildSelect = 4;
+
+        private const int FieldMinWidth = 100;
+        private const int ButtonWidth = 200;
+
+        public static void ShowWindow(int tabSelect)
+        {
+            GetWindow<PlayInstantQuickDeployWindow>("Quick Deploy");
+            toolbarInt = tabSelect;
+        }
+
+        void OnGUI()
+        {
+            toolbarInt = GUILayout.Toolbar(toolbarInt, toolbarStrings);
+            switch(toolbarInt) 
+            {
+                case BundleCreateSelect:
+                    EditorGUILayout.LabelField("AssetBundle Creation", EditorStyles.boldLabel);
+                    EditorGUILayout.LabelField("Use the Unity Asset Bundle Browser to select your game's main scene " + 
+                        "and bundle it (and its dependencies) into an AssetBundle file.", EditorStyles.wordWrappedLabel);
+                    EditorGUILayout.Space();
+                    EditorGUILayout.LabelField("AssetBundle Browser version: not found", EditorStyles.wordWrappedLabel);
```

Comment:
just so we don't lose track of this, add at the top of this method to replace all of the stub-in values here and in the other TextFields with actual computed values

Reply:
+1 to Lee's suggestion.

Specifically say:

// TODO: replace stub strings with real values

Might also do

string.Format("AssetBundle Browser version: {0}", "not found")

just to further emphasize the situation.